circle.yml

```diff
@@ -11,8 +11,12 @@ dependencies:
     - curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
     - sudo dpkg -i google-chrome.deb
     - rm google-chrome.deb
+    - sudo apt-get update
```

Comment:
Looks like we're doing `apt-get update` twice.

Reply:
Fixed.